pymchelper/writers/shieldhit.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, filename, options):
     def _header_first_line(det):
         """first line with detector geo type"""
         result = "#   DETECTOR OUTPUT\n"
+
```

Comment:
Could you replace this file with version from `master` ? Whitespace change not needed